src/ChannelServer/Network/Sending/Send.Parties.cs

```diff
@@ -0,0 +1,447 @@
+﻿// Copyright (c) Aura development team - Licensed under GNU GPL
+// For more information, see license file in the main folder
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Aura.Channel.World.Entities;
+using Aura.Shared.Network;
+using Aura.Channel.World;
+using Aura.Channel.Skills;
+using Aura.Mabi.Const;
+using Aura.Mabi.Network;
+
+namespace Aura.Channel.Network.Sending
+{
+    public static partial class Send
+    {
+        /// <summary>
+        /// Response to the party creation request, sends the client the relevant party data.
+        /// </summary>
+        /// <remarks>I feel like I'm the MSDN with that summary.</remarks>
+        /// <param name="creature"></param>
+        public static void CreatePartyR(Creature creature)
+        {
+            var packet = new Packet(Op.PartyCreateR, creature.EntityId);
+
+            var party = creature.Party;
+
+            if (creature.IsInParty)
+            {
+                packet.PutByte(1);
+
+                BuildPartyInfo(party, ref packet);
```

Comment:
The convention is to create static extension methods for Packet in the Helper namespace.